doc/state-schema.md

```diff
@@ -24,7 +24,7 @@ Each object in the `libraries` list represents a single library and has the foll
 | `source_roots`          | list   | A list of directories in the language repository where Librarian contributes code.                                                                                    | Yes      | Must not be empty, and each path must be a valid directory path. |
 | `preserve_regex`        | list   | A list of regular expressions for files and directories to preserve during the copy and remove process.                                                                    | No       | Each entry must be a valid regular expression. |
 | `remove_regex`          | list   | A list of regular expressions for files and directories to remove before copying generated code. If not set, this defaults to the `source_roots`. A more specific `preserve_regex` takes precedence. | No       | Each entry must be a valid regular expression. |
-| `release_exclude_paths` | list   | A list of directories to exclude from the release.                                                                                                                    | No       | Each entry must be a valid directory path.     |
+| `release_exclude_paths` | list   | A list of directories to exclude from the release. Files matching these paths will not be considered part of a commit for this library.                                                                                                                    | No       | Each entry must be a valid directory path.     |
```

Comment:
Perhaps change from "A list of directories" to "A list of paths" and remove the "Each entry must be a valid directory path" to "Each entry must be a valid directory file/path"?

(Basically I think we need it to be clear that it's okay to have a path to a file there. Most entries probably *will* be directories, but it's useful to have both as options.)